validator/webui/index.html

```diff
@@ -299,18 +301,35 @@
         background-size: 40px 40px;
         background-position: center left;
       }
-      .ampproject a:link { color: #0379C4; text-decoration: none; }
-      .ampproject a:visited { color: #0379C4; text-decoration: none; }
-      .ampproject a:hover { color: #0379C4; text-decoration: none; }
-      .ampproject a:active { color: #0379C4; text-decoration: none; }
+      .ampproject-right {
+        color: #0379C4;
+        margin: 10px 0;
+      }
+      .ampproject-link a:link { color: #0379C4; text-decoration: none; }
+      .ampproject-link a:visited { color: #0379C4; text-decoration: none; }
+      .ampproject-link a:hover { color: #0379C4; text-decoration: none; }
+      .ampproject-link a:active { color: #0379C4; text-decoration: none; }
```

Comment:
Since they're all doing the same thing, it's possible to combine them as:
.ampproject-link a:link,
.ampproject-link a:visited,
.ampproject-link a:hover,
.ampproject-link a:active  {
  color: #0379C4;
  text-decoration: none;
}

And I'd add the following to more closely match ampproject.org:
font-size: 14px;
font-weight: 300;
letter-spacing: 0.04em;
text-transform: uppercase;